src/Raven.Client/Documents/Session/IRevisionsSessionOperations.cs

```diff
@@ -10,6 +10,19 @@
 
 namespace Raven.Client.Documents.Session
 {
+    
+    public enum ForceRevisionStrategy
+    {
+        // Don't force revision
+        None,
```

Comment:
Why do we need None?

Reply:
Forward compatability, we may have other values there, and `Before` is an action, so we need a neutral value

Reply:
Removed - used nullable param instead

Reply:
ok - leaving None